wiki/osu!tourney/Multiplayer_Usage/en.md

```diff
@@ -0,0 +1,39 @@
+# Multiplayer Usage
+
+_Main page: [osu!tourney](/wiki/osu!tourney)_
+
+## Match creation
+
+The multipler room must be named based on the template listed in the control panel of the osu!tourney client.
+
+The template shown consists of:  
+`Acronym_in_tournament.cfg: (Team Name 1) vs (Team Name 2)`
+
+`Acronym_in_tournament.cfg` gets entirely replaced with the value of `acronym` in the `tournament.cfg` file from the [setup guide](/wiki/osu!tourney/Setup).
+If the value is defined as `Test Tourney`, the template will apear as `Test Tourney: (Team Name 1) vs (Team Name 2)`.
+
+The team names (`Team Name 1` and `Team Name 2`) can be replaced with any team name, **but keep the brackets `()` around them**.
+
+## Room management
+
+### Tournament Management Commands
+
+_See also: [Tournament Management Commands](/wiki/osu!tourney/Tournament_Management_Commands "Tournament Management Commands")_
+
+Make sure to assign the correct slots to the players joining the room using the `!mp move` and `!mp team` commands.
+
+As explained in the [spectator usage guide](/wiki/osu!tourney/Spectator_Usage), each window in the client is assigned to a slot in the multiplayer lobby. After properly setting the value of `TeamSize` in the `tournament.cfg` file, the blue team will occupy the first `TeamSize` number of slots slots and the red team will occupy the `TeamSize` number of slots below those.
+
+For example, with `TeamSize = 4`, the 1st, 2nd, 3rd and 4th slots will belong to the blue team, and the 5th, 6th, 7th and 8th slots will belong to the red team. With `TeamSize = 3`, the 1st, 2nd and 3rd slots will belong to the blue team, and the 4th, 5th and 6th slots will belong to the red team.
+
+![The correspondance of the windows in osu!tourney to the respective multiplayer room slots](Osutourneyassignment.png "osu!tourney Player Assignment")
+
+Keep in mind that osu!tourney ignores team colours in slot assignment - it only considers the slot which players in the multiplayer room occupy. Below are examples of osu!tourney with different team sizes. The numbers on the screen indicate the assigned slot in the multiplayer lobby. The numbers are not visible during normal operation and are only here for descriptional purposes:
```

Comment:
I would bold the first sentence of this paragraph given how important it is, personal thing though